### PR TITLE
Add more async

### PR DESCRIPTION
**Pull request by KSExtrez**
This PR add more async methods

### CODE REVIEW

File: src/LiteNetwork.Server/LiteNetwork.Server.csproj [hunk 1]
```diff
@@ -24,7 +24,6 @@
     <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="5.0.0" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="5.0.0" />
     <PackageReference Include="Microsoft.Extensions.ObjectPool" Version="5.0.0" />
-    <PackageReference Include="System.Buffers" Version="4.5.1" />
```
Comment (Eastrall) [inline src/LiteNetwork.Server/LiteNetwork.Server.csproj:27]: This package is not needed anymore? How does the `SocketAsyncEventArgs` buffers are allocated then?

**Review by Eastrall — COMMENTED** [review]
Nice PR! I'll merge once the dicussion about the package is solved. Thanks.

File: src/LiteNetwork.Server/LiteNetwork.Server.csproj (hunk 1 above)
Comment (KSExtrez) [reply src/LiteNetwork.Server/LiteNetwork.Server.csproj:27]: `SocketAsyncEventArgs` keep using `System.Buffers`, but it's included by default by netstandard, so we don't need an explicit reference

Comment (Eastrall) [reply src/LiteNetwork.Server/LiteNetwork.Server.csproj:27]: Alright then. Good to now that `System.Buffers` is in .NET Standard.

**Comment by Eastrall** [comment]
All good. Merging!

**Pull request merged**